<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-IBVEUFH3" failures="2" tests="2" name="grouping" time="0.157" errors="0" timestamp="21 Jul 2024 19:54:29 GMT">
  <testcase classname="qafactory.swaglabs.testCases.NegativeTestCases" name="TC01_Validate_Invalid_Username" time="0.068">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at qafactory.swaglabs.testCases.NegativeTestCases.TC01_Validate_Invalid_Username(NegativeTestCases.java:22)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- TC01_Validate_Invalid_Username -->
  <testcase classname="qafactory.swaglabs.testCases.NegativeTestCases" name="TC04_Validate_Locked_Out_User" time="0.005">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at qafactory.swaglabs.testCases.NegativeTestCases.TC04_Validate_Locked_Out_User(NegativeTestCases.java:53)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- TC04_Validate_Locked_Out_User -->
</testsuite> <!-- grouping -->
